packages/vats/test/bootstrapTests/test-vaults-integration.js

```diff
@@ -167,12 +167,12 @@ test('close vault', async t => {
       'open-vault',
     ),
     {
-      message: /^Withdrawal .* failed because the purse only contained .*/,
+      message: /^Withdrawal of .* failed because the purse only contained .*/,
     },
   );
 
   const message =
-    'Offer {"brand":"[Alleged: IST brand]","value":"[1n]"} is not sufficient to pay off debt {"brand":"[Alleged: IST brand]","value":"[5025000n]"}';
+    /Offer (\(an object\)|{"brand":"\[Alleged: IST brand\]","value":"\[1n\]"}) is not sufficient to pay off debt (\(an object\)|{"brand":"\[Alleged: IST brand\]","value":"\[5025000n\]"})$/;
```

Comment:
I wonder if this points to a missing `q(amount)` somewhere.

Reply:
Actually, it's an indicator that `@endo/ses-ava` can't unhide details when the vats are in a separate agent from the kernel.  There's not really a way to both allow censorship of important private stuff in production and also have insight into the censored stuff from another process during debugging.

Reply:
What I meant is: maybe this amount shouldn't be considered sensitive.

Reply:
Agree that the amounts shouldn't be sensible. Please quote it.